src/bindings/python/tests/test_graph/test_preprocess.py

```diff
@@ -728,3 +728,30 @@ def test_graph_set_layout_by_layout_class_thow_exception():
         layout = Layout("1-2-3D")
         ppp.input().model().set_layout(layout)
     assert "Layout name is invalid" in str(e.value)
+
+def test_pad_vector_constant_layout():
+    shape = [1, 3, 200, 200]
+    parameter_a = ops.parameter(shape, dtype=np.float32, name="A")
+    model = parameter_a
+    model = Model(model, [parameter_a], "TestModel")
+    ppp = PrePostProcessor(model)
+    ppp.input().tensor().set_shape({1, 3, 199, 199})
+    ppp.input().preprocess().pad({0, 0, 0, 0}, {0, 0, 1, 1}, 0, PaddingMode.CONSTANT)
+    assert ppp.build()
```

Comment:
Could you also check whether the output shape `model.get_output_shape(0) ` is expected?